src/views/CreateEventView.vue

```diff
@@ -14,11 +17,18 @@ import api from '../api/api';
 export default {
   name: 'CreateEvent',
   components: { EventForm },
+  data: () => ({
+    internalError: true,
```

Comment:
Leaving a note for myself that this should default to "false"

Reply:
The response statuses look good, but don't forget to address this @ZachOBrien

Reply:
thanks, just fixed